src/testdir/Make_dos.mak

```diff
@@ -72,12 +31,6 @@ report:
 		else ( echo ALL DONE )
 
 clean:
-	-del *.out
-	-del *.failed
-	-del *.res
-	-if exist $(DOSTMP) rd /s /q $(DOSTMP)
-	-if exist test.in del test.in
-	-if exist test.ok del test.ok
```

Comment:
I'm thinking whether we should remove these lines from the `clean` target right now.
These files can be left behind when someone executed the tests in an old environment. Then when they update the source code, `make clean` won't clean the files.